yaml-tests/src/test/resources/aggregate-empty-table.metrics.binpb

```diff
@@ -0,0 +1,913 @@
+�
```

Comment:
Any idea why some `.binpb` files are being displayed as a text file whereas  others are displayed as binary? It...mostly shouldn't matter, unless hypothetically, someone were to try to download this onto a `CRLF` system and the "new-lines" were helpfully changed for them. That is, assuming we're okay with checking binary data into the codebase. (I think we're mostly okay with it. It can balloon the size of git history if the files are too big or updates are too frequent, but this is...probably fine?)

Reply:
No idea why. `.binpb` is the recommendation in the `protobuf` documentation IIRC. It shows as binary in IntelliJ.